components/common/autoTeleport/AutoTeleportActionButton.vue

```diff
@@ -307,6 +309,10 @@ watch(allowAutoTeleport, (allow) => {
   if (allow) {
     autoTeleport.value = true
   }
+
+  if (!flagAutoTeleport.value) {
+    autoTeleport.value = false
+  }
```

Comment:
since 

````
const autoTeleport = ref(false)
````


```suggestion
  if (!flagAutoTeleport.value) {
    return
  }

  if (allow) {
    autoTeleport.value = true
  }
```